### PR TITLE
Better explanation for replication interval and delta interval (MultiplayerSynchronizer)

### PR DESCRIPTION
**Pull request by CrayolaEater**
Better explanation based on this: https://github.com/godotengine/godot/pull/75467

### CODE REVIEW

File: modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
 	</methods>
 	<members>
 		<member name="delta_interval" type="float" setter="set_delta_interval" getter="get_delta_interval" default="0.0">
-			Time interval between delta synchronizations. When set to [code]0.0[/code] (the default), delta synchronizations happen every network process frame.
+			Time interval between delta synchronizations. Useful when the replication is set to [code]On Change[/code]. A value of [code]0.0[/code] (the default) means that delta synchronizations happen every network process frame.
```
Comment (fire) [inline modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml:56]:
```suggestion
			Time interval between delta synchronizations. Used when the replication is set to [code]On Change[/code]. A value of [code]0.0[/code] (the default) means that delta synchronizations happen every network process frame.
```

We can't really mind read to know what is useful so I was proposing Useful -> Used.

File: modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml [hunk 2]
```diff
@@ -62,7 +62,7 @@
 			Resource containing which properties to synchronize.
 		</member>
 		<member name="replication_interval" type="float" setter="set_replication_interval" getter="get_replication_interval" default="0.0">
-			Time interval between synchronizations. When set to [code]0.0[/code] (the default), synchronizations happen every network process frame.
+			Time interval between synchronizations. Useful when the replication is set to [code]Always[/code]. A value of [code]0.0[/code] (the default) means that synchronizations happen every network process frame.
```
Comment (fire) [inline modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml:65]:
```suggestion
			Time interval between synchronizations. Used when the replication is set to [code]Always[/code]. A value of [code]0.0[/code] (the default) means that synchronizations happen every network process frame.
```

Same reasoning.

File: modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml [hunk 3]
```diff
@@ -53,7 +53,7 @@
 	</methods>
 	<members>
 		<member name="delta_interval" type="float" setter="set_delta_interval" getter="get_delta_interval" default="0.0">
-			Time interval between delta synchronizations. When set to [code]0.0[/code] (the default), delta synchronizations happen every network process frame.
+			Time interval between delta synchronizations. Used when the replication is set to [code]On Change[/code]. A value of [code]0.0[/code] (the default) means that delta synchronizations happen every network process frame.
```
Comment (Mickeon) [inline modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml:56]: I know it looks more daunting, but if you want to be accurate you could refer to the constant itself, which will redirect the user to a more proper explanation.
The existing "when set to" reads better in my opinion, maybe it should be "if" for simplicity.
```suggestion
			Time interval between delta synchronizations. Used when the replication is set to [constant SceneReplicationConfig.REPLICATION_MODE_ON_CHANGE]. If set to [code]0.0[/code] (the default), delta synchronizations happen every network process frame.
```
Alternatively, but bear in mind this is much worse for localization:
```suggestion
			Time interval between delta synchronizations. Used when the replication is set to [b]On Change[/b]. If set to [code]0.0[/code] (the default), delta synchronizations happen every network process frame.
```

File: modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml [hunk 4]
```diff
@@ -62,7 +62,7 @@
 			Resource containing which properties to synchronize.
 		</member>
 		<member name="replication_interval" type="float" setter="set_replication_interval" getter="get_replication_interval" default="0.0">
-			Time interval between synchronizations. When set to [code]0.0[/code] (the default), synchronizations happen every network process frame.
+			Time interval between synchronizations. Used when the replication is set to [code]Always[/code]. A value of [code]0.0[/code] (the default) means that synchronizations happen every network process frame.
```
Comment (Mickeon) [inline modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml:65]: Same thoughts as above
```suggestion
			Time interval between synchronizations. Used when the replication is set to [constant SceneReplicationConfig.REPLICATION_MODE_ALWAYS]. If set to [code]0.0[/code] (the default), synchronizations happen every network process frame.
```

**Comment by Mickeon** [comment]
It does not look like the issues I brought up were solved.

**Review by Mickeon — APPROVED** [review]

**Pull request merged**